pages/product/[id].vue

```diff
@@ -0,0 +1,32 @@
+<script setup>
+const route = useRoute();
+const data=Object.values(route.query);
```

Comment:
The [ID] page shouldn't use the data from the query because when entering the page from the link there is empty data